src/components/FileRepo.js

```diff
@@ -145,11 +146,26 @@ const FileRepo = ({ state, effects, ...props }) => {
                 streamData={props.streamData(props.index, props.projectId)}
               />
             </div>
+            {state.modalState.isShown && (
+              <AdvancedFacetViewModal
+                sqon={selectionSQON}
+                {...{
+                  PROJECT_ID: process.env.REACT_APP_PROJECT_ID,
```

Comment:
Just spread props and you will get projectId, index and sqon. You will need to update advanced facet to accept camelCase.
PROJECT_ID === props.projectId
ES_INDEX === props.index

API_HOST:  Inside advanced facet use utils/api. It will handle the api for you.
ES_HOST: You don't need ES_HOST in the graphql calls anymore, so you can remove this one.